scripts/generate-javadoc-components.groovy

```diff
@@ -0,0 +1,24 @@
+import groovy.json.*;
+
+// define sort order for plugins
+def keyComparator = [compare: { e1, e2 -> e1.name.compareToIgnoreCase(e2.name) }] as Comparator
+
+def components = new ArrayList<Artifact>();
```

Comment:
It would not work as you expect, because type is not defined. So i prefer explicit definition